blocks/basic/include/gnuradio-4.0/basic/And.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef AND_HPP
+#define AND_HPP
+
+#include <gnuradio-4.0/Block.hpp>
+#include <gnuradio-4.0/BlockRegistry.hpp>
+
+namespace gr::basic {
+
+GR_REGISTER_BLOCK(gr::basic::And, [uint8_t, int16_t, int32_t])
```

Comment:
OK, maybe it might be worthwhile to expand this compared to the [GR3 version](https://wiki.gnuradio.org/index.php?title=And) to cover all signed and unsigned integral types? 

@mormj what do you think? #GSoC

Reply:
Yes, I would expand this out to at least the types supported in GR3